ci/accelerator.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Command line arguments
+if [ "$ne" == '1' ]
+then
+  echo "Usage: $0 <verilator, vcs> [num_cores]"
+  exit 1
+elif [ $1 == "vcs" ]
+then
+    SUFFIX=v
+elif [ $1 == "verilator" ]
+then
+    SUFFIX=sc
+else
+  echo "Usage: $0 <verilator, vcs> [num_cores]"
+  exit 1
+fi
+
+# Default to 1 core
+N=${2:-1}
+
+cfgs=(\
+    "e_bp_multicore_1_accelerator_cfg"
+    )
+
+progs=(
+    "streaming_accelerator_loopback"
```

Comment:
Also add streaming_accelerator_demo (for the config which supports it)

Reply:
done!

Reply:
I think this is ready to merge now.

Reply:
Cool, I'll bump the CI SDK so that this can run successfully